src/ssl.c

```diff
@@ -33577,9 +33577,31 @@ int wolfSSL_DH_set0_pqg(WOLFSSL_DH *dh, WOLFSSL_BIGNUM *p,
 
     return WOLFSSL_SUCCESS;
 }
+
 #endif /* v1.1.0 or later */
 #endif /* !HAVE_FIPS || HAVE_FIPS_VERSION > 2 */
 
+/**
+ * retrive p, q and g parameter
+ * @param dh a pointer to WOLFSSL_DH
+ * @param p  a pointer to WOLFSSL_BIGNUM to be obtained dh
```

Comment:
"obtained dh" should be "obtained from dh", also applies to next 2 lines below.

Reply:
Same above.